ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -235,6 +232,8 @@ Loader {
         property string activeMessage
         readonly property bool isMessageActive: d.activeMessage === root.messageId
 
+        readonly property bool addReactionAllowed: !root.isInPinnedPopup && !root.rootStore.isUserAllowedToSendMessage
```

Comment:
Is the second one reversed? Like if you are allowed to send messages, then you can add reactions?